src/MediaBox.js

```diff
@@ -4,7 +4,15 @@ import cx from 'classnames';
 
 class MediaBox extends Component {
   componentDidMount() {
-    $('.materialboxed').materialbox();
+    if (M) {
```

Comment:
consistent check please :)

Reply:
@alextrastero Fixed this with last commit :grin: